### ISSUE TITLE
Accept metadata as cmd line arg instead of reaching for it

### ISSUE THREAD

**Issue by irinaColgiu** [wtsi-hgi/metadata-check#2]
Meta-check it should offer the possibility of accepting metadata for 1/more files from the command line in either of the following ways:
- as string
- as file .json
- streamed in from a different process
All of the above in the format that baton outputs.
In this case it shouldn't take a filepath any more as the file information is also present in the form of data_object and collection.

**Comment by colin-nolan** [comment]
You might find the Python JSON library that I've written useful for deserializing this data into the class-based Python models that you're using in metadata check:
https://github.com/wtsi-hgi/python-json

**Comment by colin-nolan** [comment]
A good place to start would be to accept metadata via stdin, supporting all of the scenarios mentioned in this ticket:
```bash
jq -n '{...}' | python metadata-check.py —flags
cat my-data-file.json | python metadata-check.py —flags
baton -c "/collection" -d "whatever.cram" | baton-list --avus | python metadata-check.py --flags
```

baton's JSON representation of objects in iRODS should be supported.

**Comment by irinaColgiu** [comment]
solved ba2f69518fc36519da53f6a823addac17ad23845
